vendor/modules.txt

```diff
@@ -131,7 +131,7 @@ github.com/gogo/protobuf/gogoproto
 github.com/gogo/protobuf/proto
 github.com/gogo/protobuf/protoc-gen-gogo/descriptor
 github.com/gogo/protobuf/sortkeys
-# github.com/golang/glog v1.2.1 => ./staging/src/github.com/golang/glog
+# github.com/golang/glog v1.2.4 => ./staging/src/github.com/golang/glog
```

Comment:
That looks suspicious to be honest. We're updating glog in `go.mod`, but we are actually using a manually vendored copy of it in `staging/src/github.com/golang/glog`.